design/ceph/object/ceph-object-store.md

```diff
@@ -38,18 +38,15 @@ spec:
 
 Now create the object store.
 ```bash
-kubectl create -f object-store.yaml
```

Comment:
These example filenames usually follow what we have in the `examples/kubernetes/ceph` folder, which don't have the `ceph` prefix since they're already in the ceph folder.